lib/eslint.js

```diff
@@ -254,13 +265,18 @@ function modifyConfigsFromComments(filename, ast, config, reportingConfig, messa
     ast.comments.forEach(function(comment) {
 
         var value = comment.value.trim();
-        var match = /^(eslint-\w+|eslint-\w+-\w+|eslint|globals?)(\s|$)/.exec(value);
+        var match = /^(eslint-\w+|eslint-\w+-\w+|eslint|exports|exported|globals?)(\s|$)/.exec(value);
 
         if (match) {
             value = value.substring(match.index + match[1].length);
 
             if (comment.type === "Block") {
                 switch (match[1]) {
+                    case "exports":
+                    case "exported":
```

Comment:
I don't think we should support both, we should pick one and stick with it. Two cases for `global` was dictated by JSHint syntax early on, I think if we would've been implementing it now, we wouldn't add two keywords for globals either.